src/class.rs

```diff
@@ -21,7 +22,7 @@ use crate::params::{
   assign_pat_to_param_def, ident_to_param_def, pat_to_param_def,
   prop_name_to_string, ts_fn_param_to_param_def,
 };
-use crate::parser::DocParser;
+use crate::swc_util::{get_location, js_doc_for_span};
```

Comment:
I am slowly trying to make the style consistent with the rest of Deno CLI, so if you could not add more that we will need to clean up later, that would be great. 😄

Reply:
Yeah, that was by accident. Sorry!